uploadNeuroDB/imaging_non_minc_insertion.pl

```diff
@@ -408,9 +417,7 @@ =head2 Methods
 }
 
 # check whether there is a candidate IDs mismatch error
-my $CandMismatchError;
-$CandMismatchError = $utility->validateCandidate($subjectIDsref);
```

Comment:
removed this as it is now part of the `determineSubjectIDs` function and is returned into `$subjectIDsref->{'CandMismatchError'}`